sos/plugins/ovn_host.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (C) 2018 Mark Michelson <mmichels@redhat.com>
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+
+
+class OVNHost(Plugin):
+    """ OVN Controller
+    """
+    plugin_name = "ovn_host"
+    profiles = ('network', 'virt')
+
+    def setup(self):
+
+        all_logs = self.get_option("all_logs")
+        limit = self.get_option("log_size")
+
+        if not all_logs:
+            self.add_copy_spec("/var/log/openvswitch/*.log",
+                               sizelimit=limit)
+        else:
+            self.add_copy_spec("/var/log/openvswitch/",
+                               sizelimit=limit)
+
+        self.add_copy_spec('/var/run/openvswitch/ovn-controller.pid')
+
+        self.add_cmd_output('ovs-ofctl -O OpenFlow13 dump-flows br-int')
+        self.add_cmd_output('ovs-vsctl list-br')
+        self.add_cmd_output('ovs-vsctl list OpenVswitch')
+
```

Comment:
Why not use a single `add_cmd_output()` call here? Since there are no differences in kwargs passed to the method you can just use an in-place list definition here.

Similarly the two `add_copy_spec()` calls around these can be merged into a single call.

Reply:
I'll make these changes as well. Thanks for the quick feedback on my patch.